src/Azure/Orleans.Transactions.AzureStorage/TransactionalState/DistributedTM/StateEntity.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+using Orleans.Transactions.Abstractions;
+using System;
+
+namespace Orleans.Transactions.DistributedTM.AzureStorage
+{
+    internal class StateEntity : TableEntity
+    {
+        public static string MakeRowKey(long sequenceId)
+        {
+            return sequenceId.ToString("x16");
```

Comment:
Please prefix rowkey and use {RKPrefix} to KRPrefix~ range pattern for min max rowkey.  This is a tested pattern we're trying to standardize on for maintainability reasons.